### ISSUE TITLE
Regarding the raw wav data

### ISSUE THREAD

**Issue by Maoshuiyang** [lagoon0o0/ToneClassifier#1]
Hi Xueyuan,

It's such an interesting project! 
I am a PhD student from CUHK. I'm writing to ask if you can share with me the raw wav data. I will appreciate a lot.
Looking forward to your reply. 

Thank you!

**Comment by alonewithyou** [comment]
@Maoshuiyang I think you could find raw data in [this](https://github.com/billryan/ToneClassifier) repository.

**Comment by Maoshuiyang** [comment]
Hi Xueyuan,

Sorry for interrupting again. There is still no raw data in the repository that you recommended, only some extracted feature files. Did you mean I can contact the author of that repository for the raw .wav data?

Thanks!

**Comment by lagoon0o0** [comment]
Yeah, I think the author of that repository probably has the raw .wav data.

**Comment by Maoshuiyang** [comment]
OK. Thank you!
